custom_components/luxtronik/translations/en.json

```diff
@@ -182,6 +182,24 @@
             },
             "efficiency_pump_minimal": {
                 "name": "Efficiency pump minimal"
+            },
+            "cooling_threshold_temperature": {
+                "name": "T_outside >"
```

Comment:
Can we find a better name?

Reply:
Yeah, this may be a bit too cryptic...
"Minimal outdoor temperature for cooling mode" ?